va/caaQuorum.go

```diff
@@ -0,0 +1,234 @@
+package va
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"sort"
+	"time"
+
+	"github.com/miekg/dns"
+	"golang.org/x/net/context"
+	"golang.org/x/net/context/ctxhttp"
+
+	"github.com/letsencrypt/boulder/metrics"
+)
+
+// We have had multiple issues resolving CAA records when the queries
```

Comment:
I would rephrase this as: We've found that a small handful of network operators block or drop CAA queries that cross their network, leading to consistent timeouts from certain network perspectives.